src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectOptions.cs

```diff
@@ -17,7 +17,7 @@
 namespace Microsoft.AspNetCore.Builder
 {
     /// <summary>
-    /// Configuration options for <see cref="OpenIdConnectOptions"/>
+    /// Configuration options for <see cref="OpenIdConnectMiddleware"/>
```

Comment:
Consistency - either use the style from CookieAuthenticationOptions, or change it to match this. Personally I like Contains the options used by X.